include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Thinking about all the previous discussion. It still won't work because of this. Did you try to tell me that earlier or did we both miss this?